app/src/main/java/app/gamenative/utils/SteamUtils.kt

```
@@ -1394,5 +1403,89 @@ object SteamUtils {
+                            intArrayOf(i), false,
+                            object : IArchiveExtractCallback {
+                                override fun getStream(index: Int, extractAskMode: ExtractAskMode): ISequentialOutStream? {
+                                    if (extractAskMode != ExtractAskMode.EXTRACT) return null
+
+                                    return ISequentialOutStream { data ->
+                                        FileOutputStream(outFile, true).use { it.write(data) }
+                                        data.size // Return the number of bytes written
+                                    }
+                                }
+
+                                override fun prepareOperation(extractAskMode: ExtractAskMode) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)